addons/mail/static/src/models/message/message.js

```diff
@@ -424,7 +424,7 @@ function factory(dependencies) {
          */
         _computeIsEmpty() {
             return (
-                (!this.body || htmlToTextContentInline(this.body) === '') &&
+                (!this.body || getTextToHTML(this.body) === '') &&
```

Comment:
I don't think this is correct, wouldn't this consider `<p></p>` as not empty? It should be empty.
Maybe you want to wrap the call to `getTextToHTML` inside the existing `htmlToTextContentInline`?